lib/build.js

```diff
@@ -148,17 +148,6 @@ function makeBundles(pathResolver, modelConfig, options, config, jsFilename, jsB
 
             let genData = gen['/data.json'](modelConfig, options);
 
-            if (options.singleFile) {
```

Comment:
So with `--single-file` option `data.json` is also writes on disk, that not so good

Reply:
But it will be wiped after the build

Reply:
And the bundles are actually written to disk too (before my changes).